### PR TITLE
GET /sinks fail with 502 Bad Gateway code when sink is yaml format

### PR DESCRIPTION
**Pull request by brnvalenca**
It was returning 502 error code because it was trying to parse a string pointer to a string in the response. So i change some stuff in the ConfigToFormat function to pass directly the address of the string to make de response object. But if i do only that, then it would crash the create sink work flow. So i set up a flag to say what kind request is beign sent (post, put, get), and based on that i choose what type of conversion inside the ConfigToFormat it must be done.

### CODE REVIEW

File: kind/values.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ orb:
       name: "orb-sinks"
       pullPolicy: "IfNotPresent"
       repository: "orbcommunity"
-      tag: "develop"
+      tag: "0.25.0-044b0c82"
```
Comment (manrodrigues) [inline kind/values.yaml:17]: why is this hard coded?

File: sinks/api/http/endpoint.go [hunk 2]
```diff
@@ -50,14 +52,19 @@ func addEndpoint(svc sinks.SinkService) endpoint.Endpoint {
 			return nil, err
 		}
 
+		// Eu posso tentar setar uma flag para sinalizar qual o tipo do método que está
```
Comment (manrodrigues) [inline sinks/api/http/endpoint.go:55]: Acho que a gente pode apagar isso aqui, ou mudar pro ingles, se for importante para voce manter o comentario :)

File: sinks/api/http/endpoint.go [hunk 3]
```diff
@@ -125,7 +132,7 @@ func updateSinkEndpoint(svc sinks.SinkService) endpoint.Endpoint {
 			return nil, err
 		}
 		var config types.Metadata
-		if req.Format != "" {
+		if req.Format == "yaml" {
```
Comment (manrodrigues) [inline sinks/api/http/endpoint.go:135]: is this modification necessary again? or your branch is not updated with develop?

**Review by manrodrigues — CHANGES_REQUESTED** [review]

File: sinks/api/http/endpoint.go (hunk 2 above)
Comment (brnvalenca) [reply sinks/api/http/endpoint.go:55]: Boa amanda, isso é garbage. vou remover!

File: kind/values.yaml (hunk 1 above)
Comment (brnvalenca) [reply kind/values.yaml:17]: its wrong, i need to return to develop. forgot again... sorry. doing right now!

**Pull request closed without merging**